.github/workflows/npm-deploy.yaml

```diff
@@ -0,0 +1,39 @@
+name: npm-deploy
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10
+          registry-url: https://registry.npmjs.org/
+
+      - name: install
+        run: |
+          npm ci
+        env:
+          CI: true
+
+      - name: build
+        run: |
+          npm run build
+        env:
+          CI: true
+
+      # - name: test
```

Comment:
should we just remove this?

Reply:
Sure. To be clear, this is only being added to the release branch so that we can use the CI to publish the package. We aren't going to merge this back into main.